app/views/layouts/application.html.haml

```diff
@@ -18,7 +18,8 @@
         %a.navbar-brand{:href => "/"} My Student Management System
         .navbar-collapse.collapse.navbar-responsive-collapse
           %ul.nav.navbar-nav
-            %li= link_to "List", "/students"
+            %li= link_to "Students", "/students"
```

Comment:
We try to use generated routes where possible, rather than hard coding urls like this. Can you instead try using `students_path` here instead.

Likewise, you should be able to use `courses_path` in the below line.

If you're not familiar with the command, you can see the routes available to you by running `bundle exec rake routes`. In the case of the first one, you should see something like:

```
students      GET      /students(.:format)      students#index
```

This tells you you can use `students_path`, which doesn't require any parameters, generates the url of `/students`, and hits the `index` action of the `students` controller.